src/resmom/mom_main.c

```diff
@@ -7110,8 +7130,27 @@ job_over_limit(job *pjob)
 	int			i;
 	u_long			limit;
 	char			*units;
+	static	resource_def	*prsdef = NULL;
+	attribute		*patresc = NULL;/* ptr to job/resv resource_list */
+	resource		*pplace = NULL;
+	int			enforce_job_wide = TRUE;
+
+	/*
+	** For enforce_on_exclhost == FALSE:
+	** Here we check to see if the job has the node "exclhost".
+	** If so, we skip enforcement on some resources.
+	*/
+	if (enforce_on_exclhost == FALSE) {
+		prsdef = find_resc_def(svr_resc_def, "place", svr_resc_size);
```

Comment:
If you are making prsdef static then you may as well want to check it for a null value and then only update it.

Reply:
done

Reply:
@vchlum sorry if my comment was unclear. I wanted to convey that since you made prsdef static, you may want to assign it a value only when it is NULL. In all other cases it will be already set to the right value.